ChangeLog

```diff
@@ -26,6 +26,7 @@
  * Removed Sample.storeToTemporaryFile|streamToRFormat methods
  * Removed GaussianProcess.GIBBS
  * Deprecated DomainIntersection|Union|DisjunctiveUnion(Domain, Domain) ctors
+ * Dropped usage of collection of Point + collection of Basis in GeneralLinearModelAlgorithm | KrigingAlgorithm
```

Comment:
* KrigingResult.getTrendCoefficients now returns a single Point
* KrigingResult.getBasisCollection was removed in favor of KrigingResult.getBasis which returns a single Basis